backend/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/headlamp-k8s/headlamp/backend
 
-go 1.24
+go 1.24.0
```

Comment:
hmm, why?

Reply:
without the minor version it fails to build/work in vscode

```
go: downloading go1.24 (linux/amd64)
go: download go1.24 for linux/amd64: toolchain not available
```